airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -2695,8 +2695,10 @@ def execute(self, operation: str, parameters: Optional[dict] = None) -> None:
         self.job_id = self.hook.run_query(sql)
 
         query_results = self._get_query_result()
-        description = _format_schema_for_description(query_results["schema"])
-        self.description = description
+        if "schema" in query_results:
+            self.description = _format_schema_for_description(query_results["schema"])
+        else:
+            self.description = []
```

Comment:
The linked issue says defaulting to None. Any reason to prefer either None or empty list?

Reply:
When I initially wrote the linked issue I thought the original behavior was `None`, but it was actually just that `self.description` was never set at all.

Now that it's a defined dictionary, I thought it would be better to set it up empty target than None.